src/main/telemetry/srxl.c

```diff
@@ -109,13 +115,9 @@ bool srxlFrameQos(sbuf_t *dst, timeUs_t currentTimeUs)
 
     sbufWriteU8(dst, SRXL_FRAMETYPE_TELE_QOS);
     sbufWriteU8(dst, SRXL_FRAMETYPE_SID);
-    sbufWriteU16BigEndian(dst, 0xFFFF); // A
-    sbufWriteU16BigEndian(dst, 0xFFFF); // B
-    sbufWriteU16BigEndian(dst, 0xFFFF); // L
-    sbufWriteU16BigEndian(dst, 0xFFFF); // R
-    sbufWriteU16BigEndian(dst, 0xFFFF); // F
-    sbufWriteU16BigEndian(dst, 0xFFFF); // H
-    sbufWriteU16BigEndian(dst, 0xFFFF); // rxVoltage
+
+    sbufFill(dst, 0xFF, 14); // Clear remainder
```

Comment:
14 should be replaced by `STRU_TELE_VTX_RESERVE_COUNT * 2`

Reply:
Nono, this is the QOS telemetry report, not the VTX report. Two different things that just happens to have a common number of 7 words and 7 bytes to fill.  But OK, I'll add symbolic names to all fillers, even though that does not enhance readability.

Reply:
I'm not advocating blind use of symbolic names, only when the value is used in more than one place.  In reviewing I thought these were the same constants, hence the request for a symbolic name. Since they are not, the symbolic name is not required unless it increases readability.